src/js/db/WDCReader.js

```diff
@@ -245,20 +247,41 @@ class WDCReader {
 					offsetMap[i] = { offset: data.readUInt32LE(), size: data.readUInt16LE() };
 			}
 
+			prevPos = data.offset;
+
 			// relationship_map
-			// ToDo: Read
-			if (header.relationshipDataSize > 0)
-				data.move((data.readUInt32LE() * 8) + 8);
+			let relationshipMap;
+
+			if (header.relationshipDataSize > 0){
+				const relationshipEntryCount = data.readUInt32LE();
+				const relationshipMinID = data.readUInt32LE(); // What are these used for?
+				const relationshipMaxID = data.readUInt32LE(); // What are these used for?

```

Comment:
I haven't looked at client code but if it's anything like older game versions, they used min ID as a base to do array-based lookups (so relationship id `X` is at `array[X - minID]`)